packages/client/components/RetroMeetingSidebar.tsx

```diff
@@ -48,24 +49,47 @@ const RetroMeetingSidebar = (props: Props) => {
   const isViewerFacilitator = facilitatorUserId === viewerId
   const isUnsyncedFacilitatorPhase = facilitatorPhaseType !== localPhaseType
   const isUnsyncedFacilitatorStage = localStage ? localStage.id !== facilitatorStageId : undefined
+  const [confirmingPhase, setConfirmingPhase] = useState<NewMeetingPhaseTypeEnum | null>(null)
   return (
     <NewMeetingSidebar
       handleMenuClick={handleMenuClick}
       toggleSidebar={toggleSidebar}
       meeting={meeting}
     >
       <MeetingNavList>
-        {phaseTypes.map((phaseType) => {
+        {phaseTypes.map((phaseType, index) => {
           const itemStage = getSidebarItemStage(phaseType, phases, facilitatorStageId)
           const {
             id: itemStageId = '',
             isNavigable = false,
-            isNavigableByFacilitator = false
-          } = itemStage || {}
+            isNavigableByFacilitator = false,
+            isComplete = false
+          } = itemStage ?? {}
           const canNavigate = isViewerFacilitator ? isNavigableByFacilitator : isNavigable
           const handleClick = () => {
-            gotoStageId(itemStageId).catch()
-            handleMenuClick()
+            const prevPhaseType = phaseTypes[index - 1]
+            const prevItemStage = prevPhaseType
+              ? getSidebarItemStage(prevPhaseType, phases, facilitatorStageId)
+              : null
+
+            const {isComplete: isPrevItemStageComplete = false, readyCount = 0} =
```

Comment:
Why do we need to check whether the previous and current stages are complete? Don't we only want to know whether the current stage is complete and, if it is, we can proceed to the next stage without showing the user the tooltip warning?

Reply:
@nickoferrall the reason is that I can navigate to the any of the previous phases and go back to the current phase.

* For example, I on the current phase which is not completed.
* I go back to some of the completed phases I should not see any tooltip
* Then I go back to the current phase that is still in progress, I shouldn't see any tooltip too.

Alternative approach is to not show a tooltip if phase is completed as is.
But also do not show a tooltip on current phase. But I need to determine the "current phase" somehow for which I didn't find implemented function, so I checking previous instead.

Reply:
-1 Logic seems flawed, I had some cases where the tooltip occurred where it shouldn't, for example clicking on the reflect phase right at the start
![Screenshot from 2022-10-10 23-15-34](https://user-images.githubusercontent.com/7331043/194953964-0fb9e5a3-6f8d-4b52-a8b7-35a00d7afff6.png)

Reply:
@Dschoordsch added a fix

Reply:
I think you could have defaulted `isPrevItemStageComplete = true` here to solve this particular case.